<launch>
<param name="/use_sim_time" value="true"/>
<rosparam file="$(find edge_brush)/config/rsparams.yaml" command="load" />
<node pkg="rosbag" name="rosbag" type="play" args="/home/eric/data/edge.bag --clock"/>
<!-- <node pkg="tf" type="static_transform_publisher" name="rslidar_base" args="0 0 0 0 0 0 /base_link /rslidar_sim 100"/> -->
<node pkg="edge_brush" name="transCloud" type="transCloud" output="screen"/>
<node pkg="edge_brush" name="groundFilter" type="groundFilter" output="screen"/>
<node pkg="edge_brush" name="rsedgeDetection" type="rsedgeDetection" output="screen"/>
<node pkg="edge_brush" name="pid_control_node" type="pidControl" output="screen"/>
<node name="rviz" pkg="rviz" type="rviz" args="-d $(find edge_brush)/config/rslidar.rviz" />
</launch>